### PR TITLE
Add pile-raycast extension

### PR DESCRIPTION
**Pull request by cr34t1ve**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/pile-raycast/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# pile-raycast Changelog
```
Comment (pernielsentikaer) [inline extensions/pile-raycast/CHANGELOG.md:1]:
```suggestion
# Pile Changelog
```

File: extensions/pile-raycast/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+        onPileCreate={handleCreate}
+      />
+      {state.posts.map((post: PilePost, index: number) => (
+        <List.Item
+          key={index}
+          icon={{ source: Icon.CircleFilled, tintColor: renderHighlightColor(post?.data?.highlight || "transparent") }}
+          title={post?.data?.title || ""}
+          actions={
+            <ActionPanel>
+              <Action.Push title="Open Post" target={<Detail markdown={post?.excerpt || ""} />} />
```
Comment (pernielsentikaer) [inline extensions/pile-raycast/src/index.tsx:253]: missing an icon

File: extensions/pile-raycast/package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "pile-raycast",
+  "title": "Pile-Raycast",
+  "description": "A raycast extension for the Pile journaling app",
```
Comment (pernielsentikaer) [inline extensions/pile-raycast/package.json:5]:
```suggestion
  "description": "A Raycast extension for the Pile journaling app",
```

File: extensions/pile-raycast/package.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "pile-raycast",
+  "title": "Pile-Raycast",
```
Comment (pernielsentikaer) [inline extensions/pile-raycast/package.json:4]:
```suggestion
  "title": "Pile",
```

File: extensions/pile-raycast/package.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  "author": "desmondsofua",
+  "categories": [
+    "Productivity",
+    "Data"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Create Post",
```
Comment (pernielsentikaer) [inline extensions/pile-raycast/package.json:16]:
```suggestion
      "title": "Create Post",
      "subtitle": "Pile",
```

File: extensions/pile-raycast/package.json [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      "description": "Template for creating a post",
```
Comment (pernielsentikaer) [inline extensions/pile-raycast/package.json:17]:
```suggestion
      "description": "Creates a post in Pile journaling application",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- The extensions suffer from [Empty state flicker](https://developers.raycast.com/basics/prepare-an-extension-for-store#empty-states)

- Remove `metadata/pile-raycast-3.png`

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by cr34t1ve** [comment]
Closed PR because the name of the extension was changed to **Mound**.
New PR can be found [here](https://github.com/raycast/extensions/pull/9351)

**Pull request closed without merging**